sql/src/test/java/io/crate/metadata/ReferenceTest.java

```diff
@@ -67,4 +93,36 @@ public void testStreaming() throws Exception {
 
         assertThat(reference2, is(reference));
     }
+
+    @Test
+    public void testStreamingWithDefaultExpression() throws Exception {
+        ReferenceIdent referenceIdent = new ReferenceIdent(tableInfo.ident(), "int_column");
+        ExpressionAnalyzer analyzer
+            = new ExpressionAnalyzer(executor.functions(),
+                                     CoordinatorTxnCtx.systemTransactionContext(),
+                                     ParamTypeHints.EMPTY,
+                                     FieldProvider.UNSUPPORTED,
+                                     null);
+        Expression expression = SqlParser.createExpression("1+1");
+        Symbol defaultExpression = analyzer.convert(expression, new ExpressionAnalysisContext());
```

Comment:
using the analyzer here to normalize an input is totally unrelated here, please remove